membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -90,7 +90,7 @@ class AttributeController(
 
   private def getLatestMobileSubscription(identityId: String)(implicit executionContext: ExecutionContext): Future[Option[MobileSubscriptionStatus]] = {
     mobileSubscriptionService.getSubscriptionStatusForUser(identityId).transform {
-      case Failure(NonFatal(error)) =>
+      case Failure(error) =>
```

Comment:
this was a non exhaustive pattern match warning,  and Futures only trap NonFatal anyway (NonFatal is only needed when using a traditional try catch block.)